eng/install-native-dependencies.sh

```diff
@@ -5,7 +5,7 @@ if [ "$1" = "Linux" ]; then
     if [ "$?" != "0" ]; then
        exit 1;
     fi
-    sudo apt install cmake llvm-3.9 clang-3.9 lldb-3.9 liblldb-3.9-dev libunwind8 libunwind8-dev gettext libicu-dev liblttng-ust-dev libcurl4-openssl-dev libssl-dev libkrb5-dev libnuma-dev
+    sudo apt install cmake gcc g++ llvm-3.9 clang-3.9 lldb-3.9 liblldb-3.9-dev libunwind8 libunwind8-dev gettext libicu-dev liblttng-ust-dev libcurl4-openssl-dev libssl-dev libkrb5-dev libnuma-dev
```

Comment:
This is not necessary. We do all build now in docker. I suggest doing the same.

Reply:
done